schema.sql

```diff
@@ -1,47 +1,30 @@
 /* Database schema to keep the structure of entire database. */
 
- create table owners (
+ create table vets(
  id int generated always as identity,
- full_name varchar(100),
+ name varchar (100),
  age int,
- primary key (id)
-);
+ date_of_graduation date,
+ primary key (id));
+
+
+ create table specializations(
+ species_id int,
+ vets_id int,
+ primary key (species_id,vets_id));
+
+ALTER TABLE public.specializations ADD CONSTRAINT specializations_fk FOREIGN KEY (species_id) REFERENCES public.species(id);
+ALTER TABLE public.specializations ADD CONSTRAINT specializations_fk_1 FOREIGN KEY (vets_id) REFERENCES public.vets(id);
+
+
+ create table visits (
+ animals_id int,
+ vets_id int,
+ date_of_visits date,
+ primary key (animals_id,vets_id,date_of_visits));
+
+
+ALTER TABLE public.visits ADD CONSTRAINT visits_fk FOREIGN KEY (animals_id) REFERENCES public.animals(id);
+ALTER TABLE public.visits ADD CONSTRAINT visits_fk_1 FOREIGN KEY (vets_id) REFERENCES public.vets(id);
+
 
-create table species (
- id int generated always as identity,
- name varchar (50),
- primary key (id)
-);
-
-Ensure the field is not null.
-alter table if exists animals alter column id set not null;
-Put the value autogenerated.
-alter table if exists animals alter column id add generated always as identity;
-ALTER TABLE IF EXISTS public.animals
-    ADD CONSTRAINT id_pk PRIMARY KEY (id);
-
-
-
-ALTER TABLE IF EXISTS animals DROP COLUMN IF EXISTS species;
-ALTER TABLE IF EXISTS animals ADD COLUMN species_id int;
-ALTER TABLE IF EXISTS animals ADD COLUMN owner_id int
-
-
-ALTER TABLE IF EXISTS animals
-    ADD CONSTRAINT fk_species FOREIGN KEY (species_id)
-    REFERENCES species (id) MATCH SIMPLE
-    ON UPDATE NO ACTION
-    ON DELETE NO ACTION
-    NOT VALID;
-CREATE INDEX IF NOT EXISTS fki_fk_species
-    ON public.animals(species_id);
-
```

Comment:
- [ ] I am not sure why these schema structures were removed but please kindly add them as most of the queries depend on them.